### ISSUE TITLE
Back button is not accessible to screen reader

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1588]
When I focus an app detail page's back button with the keyboard when using Ubuntu's built-in screen reader, it just reads, "push button" instead of "back button" or similar.

**Comment by spydon** [comment]
This is probably something that we should add directly to the `YaruBackButton`
